src/hb-ot-color-colr-table.hh

```diff
@@ -112,6 +115,48 @@ struct COLR
     return glyph_layers.length;
   }
 
+  struct accelerator_t
+  {
+    accelerator_t () {}
+    ~accelerator_t () { fini (); }
+
+    void init (hb_face_t *face)
+    {
+      colr = hb_sanitize_context_t ().reference_table<COLR> (face);
+    }
+
+    void fini ()
+    {
+      this->colr.destroy ();
+    }
+
+    bool is_valid ()
+    {
+      return colr.get_blob ()->length;
+    }
+
+    void get_related_glyphs (hb_codepoint_t glyph,
+                             hb_set_t *related_ids /* OUT */) const
+    {
+      colr->get_related_glyphs (glyph, related_ids);
+    }
```

Comment:
Feel free to use one-liner method definitions here :)
```
    void fini () { this->colr.destroy (); }

    bool is_valid () { return colr.get_blob ()->length; }

    void get_related_glyphs (hb_codepoint_t glyph,
                             hb_set_t *related_ids /* OUT */) const
    { colr->get_related_glyphs (glyph, related_ids); }
```

Reply:
Done.